exercises/concept/leap/.approaches/introduction.md

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+## General remarks
+
+- Use the remainder operator `%` to find if a number is evenly divisible by another
+- Prefer `&&` and `||` over `&` and `|` to benefit from [short-circuiting](https://docs.julialang.org/en/v1/manual/control-flow/#Short-Circuit-Evaluation). Julia will not evaluate the whole expression but only the expressions that are necessary to determine the result of the entire chain
+- You might be interested to see the Julia standard library's solution: `using Dates; @edit isleapyear(4)`
+
+## Approach: Short-circuiting
+
+```julia
+function is_leap_year(year)
+    year % 400 == 0 || year % 4 == 0 && year % 100 != 0
+end
+```
+
+If you're struggling to follow how this works, or trying to solve a similar problem, consider writing a truth table and seeing if you can work out what the simplest predicates are and how they can be combined with `&&` and `||` to give the full answer.
+
+## Mentoring notes
```

Comment:
The mentoring notes are a separate thing and should remain in the website-copy repo. The dig deeper parts are there for students, not mentors.

Could you remove this section (and also in other files, if there are mentoring notes there).